src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -229,6 +230,22 @@ public async Task<ConflictResolution> ResolveConflictsAsync()
 #if DEBUG
                     await DebugVerifyNoErrorsAsync(conflictResolution, _documentsIdsToBeCheckedForConflict).ConfigureAwait(false);
 #endif
+
+                    // Step 5: Rename declaration files
+                    if (_optionSet.GetOption(RenameOptions.RenameFile))
```

Comment:
So the philosophy of this option is if it's set, we'll always rename the containing file to match? What about API callers calling the SymbolRename APIs directly and want the same conditional logic we have in inline rename?